docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml

```diff
@@ -168,158 +190,271 @@ PLEASE NOTE: Elastic Search declares PVC(s) which might require explicit manual
+        <title>Configurando os componentes do Elastic Stack</title>
 
-      <para>Você pode precisar ou querer personalizar os componentes do
-      Elastic Stack. Os valores dos charts do componentes Elastic podem ser
-      substituídos como parte do comando de implantação do HPCC
-      Systems.</para>
+        <para>Você pode precisar ou querer personalizar os componentes da
+        Elastic Stack. Os valores dos charts dos componentes Elastic podem ser
+        substituídos como parte do comando de implantação do HPCC
+        Systems.</para>
 
-      <para>Por exemplo:</para>
+        <para>Por exemplo:</para>
 
-      <programlisting>helm install myelk hpcc/elastic4hpcclogs --set elasticsearch.replicas=2 </programlisting>
+        <programlisting>helm install myelk hpcc/elastic4hpcclogs --set elasticsearch.replicas=2 </programlisting>
 
-      <para>Consulte o repositório GitHub do Elastic Stack para obter a lista
-      completa de todas as opções do Filebeat, Elastic Search, LogStash e
-      Kibana com descrições.</para>
-    </sect2>
+        <para>Por favor, consulte o repositório GitHub do Elastic Stack para a
+        lista completa de todas as opções do Filebeat, Elastic Search,
+        LogStash e Kibana com descrições.</para>
+      </sect3>
 
-    <sect2>
-      <title>Use of HPCC Systems Component Logs in Kibana</title>
+      <sect3>
+        <title>Utilizar os componentes de logs do HPCC Systems no
+        Kibana</title>
 
-      <para>Uma vez ativado e em execução, você pode explorar e consultar os
-      logs de componentes do HPCC Systems na interface do usuário do Kibana. O
-      uso da interface do Kibana é bem suportado e documentado. Os padrões de
-      índice do Kibana são necessários para explorar os dados do Elastic
-      Search na interface do usuário do Kibana. A Elastic fornece explicações
-      detalhadas das informações necessárias para entender e utilizar
-      efetivamente a interface Elastic-Kibana. A documentação robusta do
-      Kibana deve ser consultada para obter mais informações sobre como usar a
-      interface do Kibana. Por favor, veja:</para>
+        <para>Uma vez habilitado e em execução, você pode explorar e consultar
+        os logs de componentes do HPCC Systems a partir da interface de
+        usuário do Kibana. Os padrões de índice do Kibana são necessários para
+        explorar dados do Elastic Search a partir da interface de usuário do
+        Kibana. Para mais informações sobre como usar a interface
+        Elastic-Kibana, por favor, consulte a documentação
+        correspondente:</para>
 
-      <para><ulink url="???">https://www.elastic.co/</ulink></para>
+        <para><ulink url="???">https://www.elastic.co/</ulink></para>
 
-      <para>e</para>
+        <para>e</para>
 
-      <para><ulink
-      url="???">https://www.elastic.co/elastic-stack/</ulink></para>
+        <para><ulink
+        url="???">https://www.elastic.co/elastic-stack/</ulink></para>
+      </sect3>
```

Comment:
<para><ulink
        url="???">https://www.elastic.co/elastic-stack/</ulink></para>
should be 
<para><ulink
        url="https://www.elastic.co/elastic-stack/">https://www.elastic.co/elastic-stack/</ulink></para>